tests/components/template/test_cover.py

```diff
@@ -280,6 +378,31 @@ async def test_template_state_text(
     assert text in caplog.text
 
 
+@pytest.mark.parametrize("count", [1])
+@pytest.mark.parametrize(
+    "style", [ConfigurationStyle.LEGACY, ConfigurationStyle.MODERN]
+)
+@pytest.mark.parametrize(
+    ("state_template", "expected"),
+    [
+        ("{{ 'open' }}", CoverState.OPEN),
+        ("{{ 'closed' }}", CoverState.CLOSED),
+        ("{{ 'opening' }}", CoverState.OPENING),
+        ("{{ 'closing' }}", CoverState.CLOSING),
+        ("{{ 'dog' }}", STATE_UNKNOWN),
+        ("{{ x - 1 }}", STATE_UNAVAILABLE),
+    ],
+)
+@pytest.mark.usefixtures("setup_state_cover")
+async def test_template_state_states(
```

Comment:
This test is not testing trigger cover, should it be part of this PR?

Reply:
Yes, must have missed it.